packages/base-styles/_colors.scss

```diff
@@ -8,12 +8,13 @@
 $black: #000;			// Use only when you truly need pure black. For UI, use $gray-900.
 $gray-900: #1e1e1e;
 $gray-800: #2f2f2f;
-$gray-700: #757575;		// Meets 4.6:1 text contrast against white.
-$gray-600: #949494;		// Meets 3:1 UI or large text contrast against white.
+$gray-700: #757575;	// Meets 4.6:1 text contrast against white.
+$gray-600: #949494;	// Meets 3:1 UI or large text contrast against white.
+$gray-500: #b7b7b7;	// Used for disabled controls text and icons.
```

Comment:
For what it's worth, in wp-components we have generally been using `$gray-600` for disabled text.

https://github.com/WordPress/gutenberg/blob/060d4ca57ab8441255db34e308c21a640adad2a6/packages/components/src/utils/colors-values.js#L64

Should we update all the other components to use `gray-500`? If so I will do this in a follow-up.

Reply:
Yup, that's what I've been trying to understand as well (cc @WordPress/gutenberg-design )

Reply:
Sounds reasonable. Let's take one step at a time